certbot/README.rst

```diff
@@ -101,17 +78,12 @@ Current Features
   compliant services.
 * Can get domain-validated (DV) certificates.
 * Can revoke certificates.
-* Adjustable RSA key bit-length (2048 (default), 4096, ...).
-* Adjustable `EC <https://en.wikipedia.org/wiki/Elliptic-curve_cryptography>`_
+* Adjustable EC Keys by default `EC <https://en.wikipedia.org/wiki/Elliptic-curve_cryptography>`_
   key (`secp256r1` (default), `secp384r1`, `secp521r1`).
+* Adjustable RSA key bit-length (2048 (default), 4096, ...).
```

Comment:
The way this was originally written feels odd to me. Perhaps it would be sufficient to say something like:

>Supports ECDSA (default) and RSA certificate private keys.